packages/bun/src/client.ts

```diff
@@ -30,7 +30,7 @@ export class BunClient extends ServerRuntimeClient<BunClientOptions> {
 
     const clientOptions: ServerRuntimeClientOptions = {
       ...options,
-      platform: 'bun',
+      platform: 'javascript',
```

Comment:
Yes we should definitely either add new platforms to symbolicator or converge everything. I even vote we have two platforms: `server-js` and `js` because I think that's the only differenciator. For browser js we crawl source files for node we don't.